api/lib/interfaces/controllers/security-controller.js

```diff
@@ -186,6 +187,22 @@ async function checkUserIsAdminInScoOrganizationAndManagesStudents(request, h) {
   return _replyWithAuthorizationError(h);
 }
 
+async function checkUserCanAccessCampaign(request, h) {
```

Comment:
```suggestion
async function userCanAccessCampaign(request, h) {
```
J'ai vu que toutes les méthodes étaient préfixées car check<XX>. Ce qui me fait suggérer autre chose est que, dans le cas passant `return h.response(true);`, et donc la méthode renvoie, dans l'esprit, un booléen. J'évoquerais ce cas en point tech.